subsys/net/lib/lwm2m/lwm2m_rw_cbor.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LWM2M_RW_CBOR_H_
+#define LWM2M_RW_CBOR_H_
+
+#include "lwm2m_object.h"
+
+extern const struct lwm2m_writer cbor_writer;
+extern const struct lwm2m_reader cbor_reader;
+
+size_t cbor_put_format(struct lwm2m_output_context *out,
+			     const char *format, ...);
+
+size_t cbor_put_float32fix(struct lwm2m_output_context *out,
+				 struct lwm2m_obj_path *path,
+				 float32_value_t *value);
+
+
+int do_read_op_cbor(struct lwm2m_message *msg, int content_format);
```

Comment:
Do we need to push content_format when we call lwm2m_perform_read_op() we can give LWM2M_FORMAT_APP_CBOR

Reply:
Fixed.